101-media-services-create/README.md

```diff
@@ -0,0 +1,9 @@
+# Create a Media Services Account using a template
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F101-media-services-create%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template creates a Media Services Account with its Storage account on Azure.
+
+For more information about Azure Media Services, see [Media Services Documentation](https://docs.microsoft.com/en-us/azure/media-services/).
```

Comment:
This description was based on other ones we could find on this repo: short and sweet with link to have more details. This file already defines what is going to be created and target also the main page of the AMS doc for further readings, etc.
If you are thinking about a specific sentence/block you would like I add, please let me know, otherwise my recommendation will be to let that like that. Thanks.